Apsitvarkom.UnitTests/Models/LocationTests.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Apsitvarkom.Models;
+
+namespace Apsitvarkom.UnitTests.Models;
+
+public class LocationTests
+{
+    [Test]
+    [TestCase(-91.1, 14.7, "Latitude")]
```

Comment:
`nameof(Location.Latitude)` for the third argument in the `TestCase`? You usually want to avoid hardcoding strings

Reply:
Good point. Will be changed here and in other cases.